argoverse/evaluation/competition_util.py

```diff
@@ -266,18 +255,14 @@ def save_label(argoverse_data: ArgoverseTrackingLoader, labels: List[ObjectLabel
     timestamp = argoverse_data.lidar_timestamp_list[idx]
 
     for label in labels:
+        qw, qx, qy, qz = label.quaternion
         json_data = {
             "center": {
                 "x": label.translation[0],
                 "y": label.translation[1],
                 "z": label.translation[2],
             },
-            "rotation": {
-                "x": label.quaternion[0],
-                "y": label.quaternion[1],
-                "z": label.quaternion[2],
-                "w": label.quaternion[3],
```

Comment:
@janesjanes since we express quaternions as qw, qx, qy, qz, I'm not sure this is correct here.

Since the `quaternion` package uses qw, qx, qy, qz, but we are unpacking this as qx, qy, qz, qw.